### PR TITLE
Complete non-responsible layout

### PR DESCRIPTION
**Pull request by makemesuffer**

### CODE REVIEW

File: src/styles/header.scss [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  max-width: 1123px/1674px * 100%;
+  margin: 50px auto 0;
+  display: flex;
+  align-items: center;
+}
+
+.header__burger-menu {
+  cursor: pointer;
+
+  div {
```
Comment (ilitvinovich) [inline src/styles/header.scss]: use classes instead of tags

File: src/styles/header.scss [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+}
+
+.change .header__burger-menu-line2 {
+  opacity: 0;
+}
+
+.change .header__burger-menu-line3 {
+  -webkit-transform: rotate(45deg) translate(-8px, -8px);
+  transform: rotate(45deg) translate(-8px, -8px);
+}
```
Comment (ilitvinovich) [inline src/styles/header.scss]: This should be replaced by a font icon from the icomoon.

File: src/styles/header.scss [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+.header__panel {
+  display: grid;
+  grid-template-columns: 3.8fr 2.2fr 1.2fr 4.1fr;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
```
Comment (ilitvinovich) [inline src/styles/header.scss]: do you really need this width 100%?

File: src/styles/header.scss [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+}
+
+.header__image-container {
+  position: relative;
+
+  /* не понимаю как качественно сделать эту линию */
+}
+
+.header__image {
+  position: absolute;
```
Comment (ilitvinovich) [inline src/styles/header.scss]: this shouldn't be positioned absolutely

File: src/styles/header.scss (hunk 3 above)
Comment (makemesuffer) [reply src/styles/header.scss]: done

File: src/styles/header.scss (hunk 4 above)
Comment (makemesuffer) [reply src/styles/header.scss]: done

File: src/styles/header.scss (hunk 2 above)
Comment (makemesuffer) [reply src/styles/header.scss]: done, but i have several questions

File: src/index.html [hunk 5]
```diff
@@ -3,35 +3,169 @@
...
-      </div>
-      <div class="telephone">
-      <span class="phone-number">8 800 500-61-05</span>
-      <span class="che">Круглосуточная поддержка</span>
+
+
+  <header class="header">
+    <div class="header__wrapper">
+    <nav class="header__navigation">
+      <div class="header__burger-menu" onclick="randomFunction(this)">
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/index.html]: done

File: src/index.html [hunk 6]
```diff
@@ -3,35 +3,169 @@
...
-      <a class="yazik">Ru</a>
-      </div>
-      <div class="telephone">
-      <span class="phone-number">8 800 500-61-05</span>
-      <span class="che">Круглосуточная поддержка</span>
+
+
+  <header class="header">
+    <div class="header__wrapper">
+    <nav class="header__navigation">
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/index.html]: done

File: src/index.html [hunk 7]
```diff
@@ -3,35 +3,169 @@
...
+      <div class="header__burger-menu" onclick="randomFunction(this)">
+        <div class="header__burger-menu-line1"></div>
+        <div class="header__burger-menu-line2"></div>
+        <div class="header__burger-menu-line3"></div>
       </div>
+      <div class="header__panel">
+      <a class="header__personal-area">Личный кабинет</a>
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/index.html]: done

File: src/index.html [hunk 8]
```diff
@@ -3,35 +3,169 @@
...
+      <img src="/public/assets/logo.png" alt="logo">
+      <p class="header__upper-title"> Морские прогулки</p>
+      <p class="header__lower-title">по чёрному морю</p>
+      <a class="header__button">Выбрать круиз</a>
     </div>
-    <div class="img-dashed">
-    <img src="/public/assets/dashed-line.png" alt="!!" >
     </div>
+    
+    <div class="header__image-container"><img src="/public/assets/dashed-line.png" alt="hsd" class="header__image"></div>
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/index.html]: done

File: src/index.html [hunk 9]
```diff
@@ -3,35 +3,169 @@
...
+        </svg>
+      <img src="../public/assets/first_cruise.png" alt="first_cruise">
+      </div>
+      <div class="main__text">
+      <p class="main__cruise-name">Морской круиз в Анапу</p>
+      <p class="main__route"><span class="main__text-bold">Маршрут:</span> Сочи-Лазаревское-Геленджик-Анапа</p>
+      <p class="main__time"><span class="main__text-bold">Продолжительность:</span> 10 ч. 0 мин.</p>
+      <p class="main__price">49 000 руб.</p>
+      </div>
+    </div>
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/index.html]: done

File: src/index.html [hunk 10]
```diff
@@ -3,35 +3,169 @@
...
+<div class="main__wrapper">
+  <div class="main__catalog">
+    <p>Каталог круизов</p>
+    <p>Сортировать круизы по <span>цене</span></p>
   </div>
+
+  <div class="main__grid-catalog">
+
+
+    <div class="main__container">
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/index.html]: done

File: src/index.html [hunk 11]
```diff
@@ -3,35 +3,169 @@
...
+      <div class="main__tag-container">
+        <svg class="main__tag-svg" xmlns="http://www.w3.org/2000/svg" width="89" height="101" viewBox="0 0 89 101"><g><g><path fill="#33c7fd" d="M0 0h89v101h-.017L44.5 82 .017 101H0z"/></g></g>
+        <text x="28" y="40" fill="#fff" class="main__svg-text">Хит</text>
+          <text  x="10" y="60" fill="#fff" class="main__svg-text">Продаж</text>
+        </svg>
+      <img src="../public/assets/first_cruise.png" alt="first_cruise">
+      </div>
+      <div class="main__text">
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/index.html]: done

File: src/index.html [hunk 12]
```diff
@@ -3,35 +3,169 @@
...
+  <div class="main__grid-catalog">
+
+
+    <div class="main__container">
+      <div class="main__tag-container">
+        <svg class="main__tag-svg" xmlns="http://www.w3.org/2000/svg" width="89" height="101" viewBox="0 0 89 101"><g><g><path fill="#33c7fd" d="M0 0h89v101h-.017L44.5 82 .017 101H0z"/></g></g>
+        <text x="28" y="40" fill="#fff" class="main__svg-text">Хит</text>
+          <text  x="10" y="60" fill="#fff" class="main__svg-text">Продаж</text>
+        </svg>
+      <img src="../public/assets/first_cruise.png" alt="first_cruise">
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/index.html]: done

File: src/index.html [hunk 13]
```diff
@@ -3,35 +3,169 @@
...
+      <p class="main__price">14 800 руб.</p>
+      </div>
+    </div>
+
+
+  </div>
+
+<div class="main__btn-wrapper">
+  <a class="main__extend-btn">Больше круизов</a>
+</div>
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/index.html]: net

File: src/index.html [hunk 14]
```diff
@@ -3,35 +3,169 @@
...
+         <span class="footer__container-dot"><img src="/public/assets/insta.png" alt="instagram" class="footer__dot-insta"></span>
+       </div>
+      </div>
+
+     <div>
+       <p class="footer__phone-number"><a>8 800 500-61-05</a></p>
+       <p class="footer__smaller-text"><a>Круглосуточная поддержка</a></p>
+       <p>© 2016 OOO «Городские Круизы»</p>
+     </div>
+<a></a>
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/index.html]: deleted

File: src/styles/footer.scss [hunk 15]
```diff
@@ -0,0 +1,96 @@
...
+  letter-spacing: 0.7px;
+}
+
+.footer__grid {
+  max-width: 1200px;
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: 1.3fr 2.3fr 1fr;
+
+  p:first-child {
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/styles/footer.scss]: done

File: src/styles/footer.scss [hunk 16]
```diff
@@ -0,0 +1,96 @@
...
+
+.footer__dot-ok {
+  top: 20%;
+  left: 33%;
+}
+
+.footer__dot-yt {
+  top: 20%;
+  left: 28%;
+}
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/styles/footer.scss]: added a new common css class for them.
Why can't we use position:absolute in a relative block?

File: src/styles/frequently-used.scss [hunk 17]
```diff
@@ -0,0 +1,86 @@
+@font-face {
+  font-family: "Raleway";
+  src: url("/public/fonts/Raleway-Regular.woff") format("woff"),
+  url("/public/fonts/Raleway-Bold.woff") format("woff");
+}
+
+@font-face {
+  font-family: "Raleway";
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/styles/frequently-used.scss]: deleted

File: src/styles/frequently-used.scss [hunk 18]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+@font-face {
+  font-family: "Roboto Slab";
+  src: url("https://fonts.googleapis.com/css?family=Roboto+Slab&display=swap") format("woff2");
+}
+
+$basic-font: 'Raleway', sans-serif;
+$basic-color: #0d3351;
+$secondary-font: 'Roboto Slab', sans-serif;
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/styles/frequently-used.scss]: done

File: src/styles/header.scss [hunk 19]
```diff
@@ -0,0 +1,162 @@
+@import 'frequently-used';
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/styles/header.scss]: done

File: src/styles/frequently-used.scss [hunk 20]
```diff
@@ -0,0 +1,86 @@
...
+
+body {
+  font-family: $basic-font;
+  color: $basic-color;
+  background: #f3f3f3;
```
[the remark this answers is not in the document]
Comment (makemesuffer) [reply src/styles/frequently-used.scss]: done

File: src/styles/header.scss (hunk 1 above)
Comment (makemesuffer) [reply src/styles/header.scss]: done

**Pull request merged**